src/parallel/parallel.cpp

```diff
@@ -135,6 +140,24 @@ ParallelRobinHoodHashTable::find(KeyType key)
     // TODO
 }
 
+void
+ParallelRobinHoodHashTable::add_thread_lock_manager()
+{
+    std::thread::id t_id = std::this_thread::get_id();
+    assert(!this->thread_managers_.contains(t_id) &&
+           "thread manager for current thread already exists");
+    this->thread_managers_.emplace(t_id, ThreadManager(this));
+}
+
+void
+remove_thread_lock_manager()
```

Comment:
Just noticed we're missing the parent class scope here too.

Reply:
I did this, I just forgot to push 🙄 (ugh, David)